Mobile/iOS/Templates/Release_3.4.9/xxx-thumbnails_not_downloaded.md

```diff
@@ -0,0 +1,48 @@
+###  Thumbnails for not downloaded 
+
+#### Pr: https://github.com/owncloud/ios/pull/XXX 
+
+Devices:
+
+Server:
+
+---
+
+ 
+TestID | Test Case | Steps | Expected Result | Result | Related Comment |
+:----: | :-------- | :---- | :-------------- | :----: | :-------------- |
+|**Image Formats**|||||
+|1| JPG/JPEG| 1. From web, upload a JPG file<br>2. Open the app<br>3. Tap to download the file | 2. Thumbnail is shown<br>3. Thumbnail is shown|||
+|2| PNG| 1. From web, upload a PNG file<br>2. Open the app<br>3. Tap to download the file | 2. Thumbnail is shown<br>3. Thumbnail is shown|||
+|3| GIF| 1. From web, upload a GIF file<br>2. Open the app<br>3. Tap to download the file | 2. Thumbnail is shown<br>3. Thumbnail is shown|||
+|4| TIFF| 1. From web, upload a TIFF file<br>2. Open the app<br>3. Tap to download the file | 2. Thumbnail is shown<br>3. Thumbnail is shown|||
+|**Video Formats**|||||
+|5| AVI| 1. From web, upload a AVI file<br>2. Open the app<br>3. Tap to download the file | 2. Thumbnail is shown<br>3. Thumbnail is shown|||
+|6| MPG/MPEG/MP4 | 1. From web, upload a MPG/MPEG/MP4 file<br>2. Open the app<br>3. Tap to download the file | 2. Thumbnail is shown<br>3. Thumbnail is shown|||
+|7| MOV | 1. From web, upload a MOV file<br>2. Open the app<br>3. Tap to download the file | 2. Thumbnail is shown<br>3. Thumbnail is shown|||
+|**Actions**|||||
+|8|Load files| Upload an large amount of image files in a folder|All thumbnails are correcly shown|||
+|9|Move file| 1. Upload a image<br>2. Move the image to another folder|1. Thumbnail is shown <br>2. Thumbnail is shown in the new location|||
```

Comment:
After move the image, thumbnail should be downloaded without enter to the new folder, turn wifi off and go to the new folder